pkg/apis/monitoring/v1alpha1/alertmanager_config_types.go

```diff
@@ -1002,6 +1005,98 @@ type MSTeamsV2Config struct {
 	HTTPConfig *HTTPConfig `json:"httpConfig,omitempty"`
 }
 
+// RocketChatConfig configures notifications via RocketChat.
+// It requires Alertmanager >= 0.28.0.
+type RocketChatConfig struct {
+	// Whether to notify about resolved alerts.
+	// +optional
+	SendResolved *bool `json:"sendResolved,omitempty"`
+	// The API URL for RocketChat.
+	// Defaults to https://open.rocket.chat/ if not specified.
+	// +optional
+	APIURL *URL `json:"apiURL,omitempty"`
+	// The channel to send alerts to.
+	// +optional
+	Channel *string `json:"channel,omitempty"`
+	// The sender token. Either `token` or `tokenFile` is required.
+	// +optional
+	Token v1.SecretKeySelector `json:"token,omitempty"`
+	// The file containing the sender token. Either `token` or `tokenFile` is required.
+	// +optional
+	TokenFile *string `json:"tokenFile,omitempty"`
```

Comment:
@simonpasquier do we allow for file paths to be specified? asking because it implicitly means that we will have to have access to the file